test/end-to-end/arns.test.ts

```diff
@@ -208,5 +250,7 @@ describe('ArNS', function () {
 
       assert.strictEqual(res.status, 404);
     });
+
+    // TODO: add a 402 test for undername limit exceeded
```

Comment:
I'm not sure we need it to merge, but how are you thinking we should do this?

Reply:
I can create a record on testnet ARIO contract, and add several undernames to the ANT - and use them for testing purposes

Reply:
test added in [ba0ef63](https://github.com/ar-io/ar-io-node/pull/234/commits/ba0ef6301ed876517dd14a30791918be121bda09)